### PR TITLE
chore: Fix readme linter errors for processor, aggregator, and parser plugins

### PR DESCRIPTION
**Pull request by reimda**
Use the linter in https://github.com/influxdata/telegraf/pull/10916 to clean up plugin readme markdown files.

### CODE REVIEW

File: plugins/processors/noise/README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ Depending on the function, various parameters need to be configured:
...
 ```toml @sample.conf
+The *Noise* processor is used to add noise to numerical field values. For each
+field a noise is generated using a defined probability densitiy function and
+added to the value. The function type can be configured as _Laplace_, _Gaussian_
+or _Uniform_.  Depending on the function, various parameters need to be
+configured:
+
+## Configuration
+
+```toml
```
Comment (powersj) [inline plugins/processors/noise/README.md:19]: This doesn't look right, two ```toml stanzas in a row

File: plugins/processors/noise/README.md [hunk 2]
```diff
@@ -76,5 +98,10 @@ Result of noise added to the _cpu_ metric:
 
 ```diff
 - cpu map[cpu:cpu11 host:98d5b8dbad1c] map[usage_guest:0 usage_guest_nice:0 usage_idle:94.3999999994412 usage_iowait:0 usage_irq:0.1999999999998181 usage_nice:0 usage_softirq:0.20000000000209184 usage_steal:0 usage_system:1.2000000000080036 usage_user:4.000000000014552]
-+ cpu map[cpu:cpu11 host:98d5b8dbad1c] map[usage_guest:1.0078071583066057 usage_guest_nice:0.523063861602435 usage_idle:95.53920223476884 usage_iowait:0.5162661526251292 usage_irq:0.7138529816101375 usage_nice:0.6119678488887954 usage_softirq:0.5573585443688622 usage_steal:0.2006120911289802 usage_system:1.2954475820198437 usage_user:6.885664792615023]
++ cpu map[cpu:cpu11 host:98d5b8dbad1c] map[usage_guest:1.0078071583066057
+usage_guest_nice:0.523063861602435 usage_idle:95.53920223476884
+usage_iowait:0.5162661526251292 usage_irq:0.7138529816101375
+usage_nice:0.6119678488887954 usage_softirq:0.5573585443688622
+usage_steal:0.2006120911289802 usage_system:1.2954475820198437
+usage_user:6.885664792615023]
```
Comment (powersj) [inline plugins/processors/noise/README.md:106]: If you view the rendered page this doesn't look right either. As this is code does it really need to be broken up?

File: plugins/processors/starlark/README.md [hunk 3]
```diff
@@ -223,33 +241,61 @@ def apply(metric):
...
 Open a Pull Request to add any other useful Starlark examples.
 
 [Starlark specification]: https://github.com/google/starlark-go/blob/d1966c6b9fcd/doc/spec.md
 [string]: https://github.com/google/starlark-go/blob/d1966c6b9fcd/doc/spec.md#strings
 [dict]: https://github.com/google/starlark-go/blob/d1966c6b9fcd/doc/spec.md#dictionaries
+[Starlark specification]: https://github.com/google/starlark-go/blob/master/doc/spec.md
+
+[string]: https://github.com/google/starlark-go/blob/master/doc/spec.md#strings
+
+[dict]: https://github.com/google/starlark-go/blob/master/doc/spec.md#dictionaries
```
Comment (powersj) [inline plugins/processors/starlark/README.md:301]: Aren't these duplicates?

**Review by powersj — COMMENTED** [review]
Thank you! Found couple of items

File: plugins/processors/noise/README.md (hunk 1 above)
Comment (reimda) [reply plugins/processors/noise/README.md]: Thanks. When I merged master git automatically did this and I missed it. I just pushed a fix.

File: plugins/processors/noise/README.md (hunk 2 above)
Comment (reimda) [reply plugins/processors/noise/README.md]: same as previous

File: plugins/processors/starlark/README.md (hunk 3 above)
Comment (reimda) [reply plugins/processors/starlark/README.md:298]: same as previous

**Review by powersj — APPROVED** [review]

**Pull request merged**